src/probnum/randprocs/markov/_markov.py

```diff
@@ -124,3 +100,68 @@ def _evaluate(self, x0: np.ndarray, x1: Optional[np.ndarray]) -> np.ndarray:
                 return self._markov_proc_call(args=x0).cov
 
             raise NotImplementedError
+
+
+class MarkovProcess(_MarkovBase):
+    r"""Random processes with the Markov property.
+
+    A Markov process is a random process with the additional property that
+    conditioned on the present state of the system its future and past states are
+    independent. This is known as the Markov property or as the process being
+    memoryless. A Markov process can be fully defined via an initial state and a
+    state transition.
+
+    Parameters
+    ----------
+    initarg
+        Initial starting input of the process.
+    initrv
+        Random variable describing the initial state.
+    transition
+        State transition of the system.
+
+    See Also
+    --------
+    RandomProcess : Random processes.
+    GaussianProcess : Gaussian processes.
+    """
+
+    def __init__(
+        self,
+        *,
+        initarg: np.ndarray,
+        initrv: randvars.RandomVariable,
+        transition: continuous.SDE,
+    ):
+        if not isinstance(transition, continuous.SDE):
```

Comment:
```suggestion
        if not isinstance(transition, continuous.SDE):  # pragma: no cover
```
should ignore the whole if clause in coverage